src/main/java/com/robomwm/prettysimpleshop/feature/ShowoffItem.java

```diff
@@ -80,7 +80,7 @@ private void saveCache()
         }
     }
 
-    @EventHandler
+    @EventHandler(ignoreCancelled = true, priority = EventPriority.MONITOR)
```

Comment:
Um, is this the only change in this PR? I'm confused.

Also, make a new branch or rebase to this repo so your old commits aren't included in the PR.

Reply:
some things this triggered twice

Reply:
Yea, I have noticed sometimes some items (usually doublechests near a chunk border?) can cause this "double spawn" to occur. Thanks for looking into this event, I'll probably add something to either track or look for a spawned item before having it decide to spawn the item again. (Unless this isn't what you're talking about - since I thought I included such a check in `spawnItem`)

Reply:
i just noticed when trying to debug a different issue the event would fire twice in a row for some reason, but adding that line changed it to only fire once